test/e2e/lazy_activation_policy_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	"github.com/karmada-io/karmada/test/e2e/framework"
+	testhelper "github.com/karmada-io/karmada/test/helper"
+)
```

Comment:
```suggestion
import (
	"fmt"

	"github.com/onsi/ginkgo/v2"
	appsv1 "k8s.io/api/apps/v1"
	"k8s.io/apimachinery/pkg/util/rand"

	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
	"github.com/karmada-io/karmada/test/e2e/framework"
	testhelper "github.com/karmada-io/karmada/test/helper"
)
```

Hi, perhaps this modification can solve golint's CI error report.

Reply:
Let me give it a try

Reply:
This fixed the lint error.

@chaosi-zju Can you please explain how did this worked? And what should be the order of imports?

Reply:
My IDE is `Goland`, and I configured my `Goland` as follows, it will sort my imports automatically~

![image](https://github.com/karmada-io/karmada/assets/30589999/5794a61d-0f51-4586-ab63-6ee6d1320285)

Reply:
Suggested order: standard library, system library, third-party library, this project library, different groups are separated by blank lines.